From 03db384ff0413d003de271355f59aba8c3f815a1 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Tue, 21 Apr 2020 14:30:20 +0200
Subject: [PATCH] configure.in: set AC_USE_SYSTEM_EXTENSIONS

Set AC_USE_SYSTEM_EXTENSIONS so _POSIX_SOURCE will be defined and the
following build failure will be avoided when building in c89 or c99
mode:

log.c: In function 'infomsg':
log.c:123:12: error: 'PIPE_BUF' undeclared (first use in this function)
   if (s <= PIPE_BUF) {
            ^~~~~~~~

Fixes:
 - http://autobuild.buildroot.org/results/5f4e9079b3377a869ec7002a8138b80eb6194bbb

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: sent to thomas@maier-komor.de]
---
 configure.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.in b/configure.in
index 02921c2..aba4af0 100644
--- a/configure.in
+++ b/configure.in
@@ -3,6 +3,7 @@ AC_CONFIG_HEADER(config.h)
 
 AC_CANONICAL_SYSTEM
 AC_EXEEXT
+AC_USE_SYSTEM_EXTENSIONS
 
 PACKAGE=mbuffer
 VERSION=20140310
-- 
2.25.1

